### PR TITLE
reorder self-hosted docs

### PR DESCRIPTION
**Pull request by ozRunAI**
…lm values, added section how to use custom docker registry credentials

### CODE REVIEW

**Comment by ozRunAI** [comment]
@jasonnovichRunAI we need it to be merged for 2.13-2.17

File: docs/admin/runai-setup/self-hosted/k8s/backend.md [hunk 1]
```diff
@@ -63,14 +63,15 @@ There may be cases where you need to set additional properties as follows:
...
 |<div style="width:200px"></div>| | |
 
+Use the `--set` syntax in the helm command above.  
 
+#### Custom docker registry credentials 
+To access the organization's docker registry it is required to set the registry's credentials (imagePullSecret)
 
-
-Use the `--set` syntax in the helm command above.  
+Create the secret named `runai-reg-creds` based on your existing credentials as explained [here](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#registry-secret-existing-credentials){target=_blank}.
```
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/self-hosted/k8s/backend.md:74]: @ozRunAI we try not to use the word [here] with a link.
```suggestion
Create the secret named `runai-reg-creds` based on your existing credentials. For more information, see [Create a Secret based on existing credentials](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#registry-secret-existing-credentials){target=_blank}.
```

File: docs/admin/runai-setup/self-hosted/k8s/backend.md [hunk 2]
```diff
@@ -63,14 +63,15 @@ There may be cases where you need to set additional properties as follows:
...
 | `thanos.receive.persistence.storageClass` and `postgresql.primary.persistence.storageClass` | Storage class | The installation to work with a specific storage class rather than the default one |
-| `global.imagePullSecrets:` <br> &ensp; `- name: <secret-name>`  | Docker secret | Provide credentials for accessing the organization's docker registry. This is required for air-gapped environments  |
 | `<component>` <br> &ensp;`resources:` <br> &emsp; `limits:` <br> &emsp; &ensp; `cpu: 500m` <br> &emsp; &ensp; `memory: 512Mi` <br> &emsp; `requests:` <br> &emsp; &ensp; `cpu: 250m` <br> &emsp; &ensp; `memory: 256Mi`  | Pod request and limits  |  `<component>` may be anyone of the following: `backend`, `frontend`, `assetsService`, `identityManager`, `tenantsManager`, `keycloakx`, `grafana`, `authorization`, `orgUnitService`,`policyService`  |   
 |<div style="width:200px"></div>| | |
 
+Use the `--set` syntax in the helm command above.  
 
+#### Custom docker registry credentials 
+To access the organization's docker registry it is required to set the registry's credentials (imagePullSecret)
```
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/self-hosted/k8s/backend.md:72]: @ozRunAI missing period at the end of the sentence.
```suggestion
To access the organization's docker registry it is required to set the registry's credentials (imagePullSecret).
```

File: docs/admin/runai-setup/self-hosted/k8s/backend.md [hunk 3]
```diff
@@ -48,7 +48,7 @@ Perform the instructions for [local certificate authority](../../config/org-cert
 
 
 ## (Optional) Additional Configurations
-
+#### Helm values
```
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/self-hosted/k8s/backend.md:51]: @ozRunAI is this supposed to be a H4? Maybe you meant an H3?
```suggestion
### Helm values
```

File: docs/admin/runai-setup/self-hosted/ocp/backend.md [hunk 4]
```diff
@@ -43,6 +43,7 @@ Perform the instructions for [local certificate authority](../../config/org-cert
 
 ## (Optional) Additional Configurations
 
+#### Helm values
```
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/self-hosted/ocp/backend.md:46]: @ozRunAI, same as above.

File: docs/admin/runai-setup/self-hosted/ocp/backend.md [hunk 5]
```diff
@@ -55,9 +56,16 @@ There may be cases where you need to set additional properties as follows:
...
 | `thanos.receive.persistence.storageClass` and `postgresql.primary.persistence.storageClass` | Storage class | The installation to work with a specific storage class rather than the default one |
-| `global.imagePullSecrets:` <br> &ensp; `- name: <secret-name>`  | Docker secret | Provide credentials for accessing the organization's docker registry. This is required for air-gapped environments  |
 | `<component>` <br> &ensp;`resources:` <br> &emsp; `limits:` <br> &emsp; &ensp; `cpu: 500m` <br> &emsp; &ensp; `memory: 512Mi` <br> &emsp; `requests:` <br> &emsp; &ensp; `cpu: 250m` <br> &emsp; &ensp; `memory: 256Mi`  | Pod request and limits  |  `<component>` may be anyone of the following: `backend`, `frontend`, `assetsService`, `identityManager`, `tenantsManager`, `keycloakx`, `grafana`, `authorization`, `orgUnitService`,`policyService`  |   
-|<div style="width:200px"></div>| | |Use the `--set` syntax in the helm command above.  
+|<div style="width:200px"></div>| | |
+
+Use the `--set` syntax in the helm command above.  
+
+#### Custom docker registry credentials 
+To access the organization's docker registry it is required to set the registry's credentials (imagePullSecret)
```
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/self-hosted/ocp/backend.md:65]: @ozRunAI same as above.

File: docs/admin/runai-setup/self-hosted/ocp/backend.md [hunk 6]
```diff
@@ -55,9 +56,16 @@ There may be cases where you need to set additional properties as follows:
...
+
+Create the secret named `runai-reg-creds` based on your existing credentials as explained [here](https://docs.openshift.com/container-platform/latest/openshift_images/managing_images/using-image-pull-secrets.html#images-allow-pods-to-reference-images-from-secure-registries_using-image-pull-secrets){target=_blank}.
```
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/self-hosted/ocp/backend.md:67]: @ozRunAI same as above.

File: docs/admin/runai-setup/self-hosted/k8s/backend.md [hunk 7]
```diff
@@ -1,20 +1,29 @@
 
 # Install the Run:ai Control Plane 
 
-
-## Domain certificate
+## Prerequisites 
+### Domain certificate
```
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/self-hosted/k8s/backend.md:5]: @ozRunAI needs a line between headings

```suggestion

### Domain certificate
```

File: docs/admin/runai-setup/self-hosted/k8s/backend.md [hunk 8]
```diff
@@ -63,29 +70,31 @@ There may be cases where you need to set additional properties as follows:
...
 |<div style="width:200px"></div>| | |
 
+Use the `--set` syntax in the helm command above.  
 
 
 
-Use the `--set` syntax in the helm command above.  
 
+## Next Steps
 ### Connect to Run:ai User Interface
```
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/self-hosted/k8s/backend.md:82]:
```suggestion

### Connect to Run:ai User Interface
```

**Review by yarongol — CHANGES_REQUESTED** [review]

**Pull request merged**